README.md

```diff
@@ -4,22 +4,116 @@
 [![r2c Twitter](https://img.shields.io/twitter/follow/r2cdev?label=Follow%20r2cdev&style=social&color=blue)](https://twitter.com/intent/follow?screen_name=r2cdev)
 
-`semgrep` is a tool for easily detecting and preventing bugs and anti-patterns in
-your codebase. It combines the convenience of `grep` with the correctness of
-syntactical and semantic search. Developers, DevOps engineers, and security engineers
-use `semgrep` to write code with confidence.
+<p align="center">
+  <a href="#overview">Overview</a>
+  <span> · </span>
+  <a href="#installation">Installation</a>
+  <span> · </span>
+  <a href="#usage">Usage</a>
+  <br/>
+  <a href="#resources">Resources</a>
+  <span> · </span>
+  <a href="#contributing">Contributing</a>
+  <span> · </span>
+  <a href="#commercial-support">Commercial Support</a>
+</p>
+
+<h3 align="center">
+  A fast, lightweight static analysis tool that lets you search code like you’d write it.
+</h3>  
+<h3 align="center">
+  Find security bugs. Enforce coding standards. Scan every PR.
+</h3>
+<h3 align="center">  
+  Easily block bad code so you can get back to building the next great thing.
+</h3>
+
+Semgrep is a tool for easily detecting and preventing bugs and anti-patterns
+in your codebase. It combines the convenience and rapid iteration speed of
+`grep`, but is "code-aware"; that is, you can easily match function calls, class
+or method definitions, and more.
+
+## Quickstart
+
+After [installing semgrep](#installation), run the default set of checks with:
+
+```bash
+semgrep --config=r2c path/to/repo
+```
 
-**Try it now:** [https://semgrep.live](https://semgrep.live/)
+Or try Semgrep in your browser: [https://semgrep.live](https://semgrep.live/).
 
 ## Overview
 
-Language support:
+### Key features
+
+* **Language aware**: Semgrep parses source code into Abstract Syntax Trees (ASTs), so it understands function calls, method and class definitions, conditionals, and more. 
+  * Unlike regexes, Semgrep won't get tripped up matching things you don't care about in comments or string literals.
+* **Fast**: Speedy enough to run on every build, commit, or file save.
+* **Batteries included**: Leverage hundreds of preexisting security checks for popular languages and frameworks in the community [registry](https://semgrep.live/r), including coverage for the [OWASP Top 10](https://owasp.org/www-project-top-ten/).
+* **Handles multiple languages**: No need to learn and maintain multiple tools for your polyglot environment (e.g. ESLint, find-sec-bugs, rubocop, gosec, ...).
+* **Easy to tweak**: Rules look like the code you’re searching, no static analysis PhD required.
+* Does **not** require compilation, Semgrep runs on source code directly. 
```

Comment:
Perhaps another way to phrase this that better matches the pattern of the other bullets:

* **Runs on source code**: Semgrep doesn’t require compilation because it runs on source directly.

Reply:
This is a big deal for Java. Thoughts on calling it out specifically?